CHANGES.md

```diff
@@ -3,9 +3,11 @@
 # Changes #
 
 * @dev (????-??-??)
-  * Added TTL 74182: look-ahead carry generator
-  * Added TTL 74181: arithmetic logic unit
+  * Added TTL 74182: look-ahead carry generator.
+  * Added TTL 74181: arithmetic logic unit.
   * Fixed Karnaugh map color index bug
```

Comment:
I think that we also have to put a . here (was my mistake in my previous PR)

Reply:
I apparently overlooked that one. Fixed